src/uct/ib/dc/dc_mlx5.h

```diff
@@ -37,12 +37,11 @@ struct ibv_ravh {
 #  define UCT_DC_RNDV_HDR_LEN   0
 #endif
 
-#define UCT_DC_MLX5_IFACE_MAX_USER_DCIS 15
+#define UCT_DC_MLX5_IFACE_MAX_USER_DCIS 128
 #define UCT_DC_MLX5_KEEPALIVE_NUM_DCIS  1
 #define UCT_DC_MLX5_IFACE_MAX_DCI_POOLS 8
-#define UCT_DC_MLX5_IFACE_MAX_DCIS      ((UCT_DC_MLX5_IFACE_MAX_USER_DCIS * \
-                                          UCT_DC_MLX5_IFACE_MAX_DCI_POOLS) + \
-                                          UCT_DC_MLX5_KEEPALIVE_NUM_DCIS)
+#define UCT_DC_MLX5_IFACE_MAX_DCIS(_ndci, _npool) (((_ndci) * (_npool)) + \
```

Comment:
now that we allocate DCIs dynamically, this no longer need to be a macro.
it can be calculated directly before using ucs_calloc

Reply:
done